client/src/main/java/org/eclipse/hono/client/Command.java

```diff
@@ -84,6 +85,8 @@ public static Command from(
         Objects.requireNonNull(tenantId);
         Objects.requireNonNull(deviceId);
 
+        final Message commandMessage = HonoProtonHelper.getMessageCopyWithCopiedProperties(message);
```

Comment:
I think it would be good to also have a note in the javadoc, that this method will return a new instance.

Reply:
I've adapted the javadoc of the `from` method accordingly.